src/qat/purr/backends/qblox/device.py

```diff
@@ -183,6 +186,27 @@ def _prepare_config(self, package: QbloxPackage, sequencer: Sequencer):
 
         return qblox_config
 
+    def reset(self, hard=False):
+        """
+        TODO - Needs a rethink when daisy-chaining clusters
+        """
+
+        self._resources.clear()
+
+        if hard:
```

Comment:
can we add a comment in the doc string on the difference between a hard reset and a soft reset